packages/@aws-cdk/aws-appsync/test/appsync.test.ts

```diff
@@ -73,3 +74,31 @@ test('when xray is enabled should not throw an Error', () => {
     XrayEnabled: true,
   });
 });
+
+test('appsync GraphqlApi should be configured with custom CW Logs Role ARN when specified', () => {
+  // WHEN
+  new appsync.GraphqlApi(stack, 'api-custom-cw-logs-role', {
+    authorizationConfig: {},
+    name: 'api',
+    schema: appsync.Schema.fromAsset(path.join(__dirname, 'appsync.test.graphql')),
+    logConfig: {
+      roleArn: 'testCustomRoleArn',
+    },
+  });
+
+  // THEN
+  expect(stack).toHaveResourceLike('AWS::AppSync::GraphQLApi', {
+    LogConfig: {
+      CloudWatchLogsRoleArn: 'testCustomRoleArn',
+    },
+  });
+});
+
+test('appsync GraphqlApi should not use custom role for CW Logs when not specified', () => {
+  // EXPECT
+  expect(stack).not.toHaveResourceLike('AWS::AppSync::GraphQLApi', {
+    LogConfig: {
+      CloudWatchLogsRoleArn: 'testCustomRoleArn',
+    },
+  });
+});
```

Comment:
I believe a better test would be to ensure that the default role is created

Reply:
Yeah, good call. I'll check for default role being created